generator/views/lookml_utils.py

```diff
@@ -65,20 +67,33 @@ def _get_dimension(path: Tuple[str, ...], field_type: str, mode: str) -> Dict[st
         if path in MAP_LAYER_NAMES:
             result["map_layer_name"] = MAP_LAYER_NAMES[path]
     result["name"] = "__".join(name)
+    if name[0] == "metrics":
```

Comment:
We could probably cover client_info metrics quite easily with this approach too. It is admittedly a huge hack job.